code/modules/urist/modules/ghost_powers.dm

```diff
@@ -11,10 +11,11 @@
 
 /obj/structure/window/attack_ghost(mob/observer/ghost/user as mob)
 	. = ..()
-	playsound(src.loc, 'sound/effects/glassknock.ogg', 80, 1)
-	user.visible_message("Something knocks on the [src.name].",
-							"You knock on the [src.name].",
-							"You hear a knocking sound.")
+	if(round_is_spooky())
```

Comment:
TBH, rather than add those overrides, I probably should have just defined /atom/proc/spooky() and make attack_ghost() call that after this check, would save us the boilerplate. Would auto-examine when you wanted to spoop, but still.